tidy3d/web/heat.py

```diff
@@ -0,0 +1,366 @@
+"""Web API for heat simulation"""
```

Comment:
We'll clearly need to unify the webapis between all of the solvers because this is almost a direct copy of the `Simulation` webapi right? 

I'm working with MC to provide a simple interface where they provide a template for a `Submittable` task (simulation, mode solver, heat simulation) and we implement this template in our webapi and use a single webapi to manage it. We'll see how that looks but that would hopefully eliminate the need for this file.

Reply:
let's ignore web api stuff for now until stub is merged